README.md

```diff
@@ -23,11 +24,13 @@ the command is targeted at one dataset and impacts it and all its children,
 allowing to get a consistent point-in-time set of snapshots across multiple
 datasets.
 
-Zetup Inztructionz
+Compilation Instructions
```

Comment:
that would be

Complilation Inztructionz

Reply:
I think that spelling what was the original PR aimed to "fix", but fine with me ;)

Reply:
:-) humorlesbunch